src/poll_manager.ts

```diff
@@ -101,6 +101,9 @@ export class PollManager<SCG extends ExtendableGenerics = DefaultGenerics> {
   };
 
   private setOrOverwriteInCache = (pollResponse: PollResponse<SCG>, overwriteState?: boolean) => {
+    if (this.client.options.disableCache) {
```

Comment:
@isekovanic would this be necessary also for ThreadManager?

Reply:
No, for `ThreadManager` you have to use the `queryThreads` directly from it rather than `client.queryThreads` - so there is an alternative. For polls there isn't much of a choice hence why I added it